src/components/RelocationBox/RelocationBox.jsx

```diff
@@ -18,7 +18,7 @@ import { renderEmail } from 'react-html-email';
 import PropTypes from 'prop-types';
 import RelocateBoxIcon from '../BoxIcons/RelocateBoxIcon.svg';
 import SaveChangesIcon from '../BoxIcons/SaveChangesIcon.svg';
-import { FYABackend, sendEmail } from '../../common/utils';
+import { FYABackend, getLatLon, sendEmail } from '../../common/utils';
```

Comment:
nit: Can we rename this to getLatLong? I've always seen "long" used as short for "longitude". Please change any other uses of "lon" to "long" as well.

Reply:
done!